final/src/main/java/ua/epam/spring/hometask/converter/AirDateConverter.java

```diff
@@ -3,5 +3,21 @@
 import ua.epam.spring.hometask.domain.AirDate;
 import ua.epam.spring.hometask.dto.AirDateDTO;
 
+import java.util.Collection;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 public interface AirDateConverter extends Converter<AirDate, AirDateDTO> {
+
+    default Set<AirDateDTO> convertToDTOSet(final Collection<AirDate> entities) {
+        return entities.stream()
+                .map(this::convertToDTO)
+                .collect(Collectors.toSet());
+    }
```

Comment:
The aim of the converter is to transform one object to another. Move this logic to the proper converter.